### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by z-a-f**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#23148 Documentation cleanup**

Differential Revision: [D16414202](https://our.internmc.facebook.com/intern/diff/D16414202)

### CODE REVIEW

File: torch/nn/quantized/modules/conv.py [hunk 1]
```diff
@@ -14,9 +14,48 @@
 
 
 class Conv2d(_ConvNd):
+    r"""Applies a 2D convolution over a quantized input signal composed of
+    several quantized input planes.
+
+    For details on input arguments, parameters, and implementation see
+    :class:`~torch.nn.quantized.Conv2d`.
```
Comment (soumith) [inline torch/nn/quantized/modules/conv.py]: you mean `torch.nn.Conv2d` here, but you wrote `quantized` which is circular reference

File: torch/nn/quantized/modules/conv.py [hunk 2]
```diff
@@ -14,9 +14,48 @@
...
+
+    .. note::
+        Only `zeros` is supported for the :attr:`padding_mode` argument.
+
+    .. note::
+        Only `torch.quint8` is supported for the input data type.
+
+    .. note::
+        The :attr:`weight` is stored as a packed tensor, and can be accessed
+        using :attr:`_packed_weight` attribute.
```
Comment (soumith) [inline torch/nn/quantized/modules/conv.py]: does this mean this module doesn't have `.weight` present? the language doesn't make that clear, or give the user an understanding of why `.weight` is simply not used and a new attribute name is introduced

File: torch/nn/quantized/functional.py [hunk 3]
```diff
@@ -5,17 +5,61 @@
...
-add_relu = ops.quantized.add_relu
+def relu(input, inplace=False):
+    # type: (Tensor, bool) -> Tensor
+    r"""relu(input, inplace=False) -> Tensor
 
+    .. note::
+      :attr:`inplace` is not supported for the quantized relu.
+
+    Applies the rectified linear unit function element-wise. See
+    :class:`~torch.nn.quantized.ReLU` for more details.
```
Comment (soumith) [inline torch/nn/quantized/functional.py]: again, `torch.nn.functional.ReLU` is what you meant to type here

**Review by soumith — CHANGES_REQUESTED** [review]

File: torch/nn/quantized/modules/conv.py (hunk 2 above)
Comment (zafartahirov) [reply torch/nn/quantized/modules/conv.py]: Oh, yea -- this is confusing. The `weight` still exists -- I guess I should remove the `_packed_weight` from the docs.

File: torch/nn/quantized/functional.py (hunk 3 above)
Comment (zafartahirov) [reply torch/nn/quantized/functional.py]: Actually, about that -- don't we want to refer the functional to its module counterpart? The documentation in the `torch.nn.quantized.ReLU` is more relevant to this than the non-quantized counterpart

**Review by soumith — APPROVED** [review]

**Pull request closed without merging**